pandas/tests/indexes/test_indexing.py

```diff
@@ -0,0 +1,84 @@
+"""
+test_indexing tests the following Index methods:
+    __getitem__
+    get_loc
+    get_value
+    __contains__
+    take
+    where
+    get_indexer
+    slice_locs
+    asof_locs
+
+The corresponding tests.indexes.[index_type].test_indexing files
+contain tests for the corresponding methods specific to those Index subclasses.
```

Comment:
why do we not have the tests here parametrised for the different Index types?

Reply:
there are *lots* of tests that are not correctly paramateerized yet